avconv.js

```diff
@@ -160,7 +221,7 @@ module.exports = function avconv(params) {
 
     avconv.on(eventType, function(exitCode, signal) {
         stream.end();
-        stream.emit('exit', exitCode, signal);
+        stream.emit('exit', exitCode, signal, parseMetaData(output));
```

Comment:
what if i am not interested in meta data in my app? can this be suppressed to gain speed?